PrivateHeaders/APPX/ZIP.h

```diff
@@ -325,6 +372,9 @@ namespace appx {
            << "xmlns=\"http://schemas.microsoft.com/appx/2010/blockmap\" "
            << "HashMethod=\"http://www.w3.org/2001/04/xmlenc#sha256\">";
         for (const ZIPFileEntry &entry : otherEntries) {
+            if (_IsAPPXFile(entry.fileName)) {
```

Comment:
Microsoft doesn't provide the appx files in the AppxBlockMap.xml, so that is why we don't do that either. They are stored in [Content_Types].xml so we have them in other entries.